lib/runtime-core/src/sys/unix/memory.rs

```diff
@@ -238,13 +249,19 @@ impl Clone for Memory {
     }
 }
 
+/// Kinds of memory protection.
 #[derive(Serialize, Deserialize, Debug, Copy, Clone, PartialEq, Eq)]
 #[allow(dead_code)]
 pub enum Protect {
+    /// No protection.
     None,
+    /// Read protection.
     Read,
+    /// Read/write protection.
```

Comment:
I think this description is ambiguous (does "protection" refer to what is forbidden, so this is execute-only memory?). What it means is "read and write are permitted" with the implication that execution is not. It should say that. Suggestion: "Read and write only"

Reply:
Updated these with your suggestion.